containers/server-image/Dockerfile

```diff
@@ -9,25 +9,24 @@ ARG PRODUCT_PATTERN="patterns-uyuni_server"
 
 # Add distro and product repos
 COPY add_repos.sh /usr/bin
-RUN sh add_repos.sh ${PRODUCT_REPO}
+RUN sh add_repos.sh
 
 # Copy Uyuni setup script
 COPY setup_env.sh /root/setup_env.sh
 COPY uyuni-setup.service /usr/lib/systemd/system/
 
 # Main packages
+COPY remove_unused.sh .  
+RUN echo "rpm.install.excludedocs = yes" >>/etc/zypp/zypp.conf
 # TODO Remove vi and possibly python3-simplejson
 RUN zypper ref
-RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses  \
+RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses --force-resolution \
```

Comment:
Why do we need `--force-resolution` ? Those should be handled at OBS project config level